learnrl/playground.py

```diff
@@ -306,3 +308,38 @@ def test(self, episodes, **kwargs):
                 UserWarning
             )
         self.run(episodes, render=render, learn=learn, verbose=verbose, **kwargs)
+
+    def set_agents_order(self, agents_order: list) -> list:
+        """Change the agents_order.
+
+        This will change what agent is described by the results of `env.turn`
+        if the environment is subclassing :class:`~learnrl.envs.TurnEnv`.
+
+        Args:
+            agents_order: New agents order. Default is range(n_agents).
```

Comment:
I don't really understand the first paragraph.

And maybe you should indicate that it is the list of the agents indices?